RecoEgamma/EgammaPhotonProducers/python/conversionTrackSequence_cff.py

```diff
@@ -9,8 +9,7 @@
 from RecoEgamma.EgammaPhotonProducers.ckfOutInTracksFromConversions_cfi import *
 from RecoEgamma.EgammaPhotonProducers.ckfInOutTracksFromConversions_cfi import *
 
-ckfTracksFromConversionsTask = cms.Task(conversionTrackCandidates,ckfOutInTracksFromConversions,ckfInOutTracksFromConversions)
```

Comment:
Why did you remove the intermediate Task?

Reply:
Because I do not want it in the new sequence ! The sequence for finding cconversions from lowPtEleTracks does not need that.

Reply:
Addendum. What I said is valid for the new Open sequence. Your comment is on the original one. So again, let me check with care